Software/bsw/gen/CanIf_Cfg.h

```diff
@@ -304,25 +306,26 @@ CanIfTxPduUserTrigger- TransmitName must be PduR_CanIfTriggerTransmit.
 
 [SWS_CANIF_00891] Configuration of <User_TriggerTransmit>(): If CanIfTxPduUserTxConfirmationUL is set to CDD,
 the name of the API <User_TriggerTransmit>() has to be configured via parameter CanIfTxPdu- UserTriggerTransmitName. */
-#if(CANIF_TX_PDU_TRIGGER_TRANSMIT==STD_ON)
-	#if(CanIfTxPduUserTxConfirmationUL==PDUR)
-		#define CANIF_TX_PDU_USER_TRIGGER_TRANSMIT_NAME	PduR_CanIfTriggerTransmit
-	#elif(CanIfTxPduUserTxConfirmationUL==CAN_NM)
-		#define CANIF_TX_PDU_USER_TRIGGER_TRANSMIT_NAME	CanNm_CanIfTriggerTransmit
-	#elif(CanIfTxPduUserTxConfirmationUL==J1939NM)
-		#define CANIF_TX_PDU_USER_TRIGGER_TRANSMIT_NAME	J1939Nm_CanIfTriggerTransmit
-	#elif(CanIfTxPduUserTxConfirmationUL==J1939TP)
-		#define CANIF_TX_PDU_USER_TRIGGER_TRANSMIT_NAME	J1939Tp_CanIfTriggerTransmit
-	#elif(CanIfTxPduUserTxConfirmationUL==CAN_TP)
-		#define CANIF_TX_PDU_USER_TRIGGER_TRANSMIT_NAME	CanTp_CanIfTriggerTransmit
-	#elif(CanIfTxPduUserTxConfirmationUL==XCP)
-		#define CANIF_TX_PDU_USER_TRIGGER_TRANSMIT_NAME	Xcp_CanIfTriggerTransmit
-	#elif(CanIfTxPduUserTxConfirmationUL==CAN_TSYN)
-		#define CANIF_TX_PDU_USER_TRIGGER_TRANSMIT_NAME	CanTSyn_CanIfTriggerTransmit
-	#elif(CanIfTxPduUserTxConfirmationUL==CDD)
-		#define CANIF_TX_PDU_USER_TRIGGER_TRANSMIT_NAME	Cdd_CanIfTriggerTransmit /* Selectable */
-	#endif	
-#endif
+/* CanIfTxPduUserTxConfirmationUL is implemented as PostBuild Configuration */
+// #if(CANIF_TXPDU_TRIGGERTRANSMIT==STD_ON)
+	// #if(CanIfTxPduUserTxConfirmationUL==PDUR)
+		// #define CANIF_TX_PDU_USER_TRIGGER_TRANSMIT_NAME	PduR_CanIfTriggerTransmit
+	// #elif(CanIfTxPduUserTxConfirmationUL==CAN_NM)
+		// #define CANIF_TX_PDU_USER_TRIGGER_TRANSMIT_NAME	CanNm_CanIfTriggerTransmit
+	// #elif(CanIfTxPduUserTxConfirmationUL==J1939NM)
+		// #define CANIF_TX_PDU_USER_TRIGGER_TRANSMIT_NAME	J1939Nm_CanIfTriggerTransmit
+	// #elif(CanIfTxPduUserTxConfirmationUL==J1939TP)
+		// #define CANIF_TX_PDU_USER_TRIGGER_TRANSMIT_NAME	J1939Tp_CanIfTriggerTransmit
+	// #elif(CanIfTxPduUserTxConfirmationUL==CAN_TP)
+		// #define CANIF_TX_PDU_USER_TRIGGER_TRANSMIT_NAME	CanTp_CanIfTriggerTransmit
+	// #elif(CanIfTxPduUserTxConfirmationUL==XCP)
+		// #define CANIF_TX_PDU_USER_TRIGGER_TRANSMIT_NAME	Xcp_CanIfTriggerTransmit
+	// #elif(CanIfTxPduUserTxConfirmationUL==CAN_TSYN)
+		// #define CANIF_TX_PDU_USER_TRIGGER_TRANSMIT_NAME	CanTSyn_CanIfTriggerTransmit
+	// #elif(CanIfTxPduUserTxConfirmationUL==CDD)
+		// #define CANIF_TX_PDU_USER_TRIGGER_TRANSMIT_NAME	Cdd_CanIfTriggerTransmit /* Selectable */
+	// #endif	
+// #endif
```

Comment:
Yes these changes are **CORRECT** as all of these parameters are implemented in the PostBuild files and they are mistakenly not commented (we have pushed this file without commenting these lines as these parameters has been changed to be postbuild not pre-compile)
**BUT** we are commenting them in CanIf_Lcfg_PBcfg Branch and will be pushed for the whole teams.

Reply:
Well . I will merge your branch again when you do these changes and get rid of the wrong formulas which are repeated and not mentioned in AUTOSAR SWS